### PR TITLE
🔧  MAINT - Update github content

### LINKED ISSUE #5: [MAINT] - Add README.md
This should be a top-level readme for the project 

## Tasks
To be defined

### LINKED ISSUE #6: [MAINT] - Add default community health files to this repo
This should host all the community health docs for the repos across the org:

1. Code of Conduct
2. Contributing
3. Security (need to define this)

### PR DESCRIPTION
**Pull request by trallard**
- :wrench: Update templates
- :wrench: Add PR template
- 🚨 Add pre-commit to keep files consistent

## Reference Issues/PRs

Fixes: #6  
Fixes: #5 

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [x] Other (please describe): update templates (add links, improve wording, and fix 

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

## Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

## Non-text content

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense and no information is missing.

### CODE REVIEW

**Review by HarshCasper — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
 # .github
-✨ Default community health files for all repos in the Nebari-dev organisation
+
+This is a meta repository that defines some shared files that the repositories for the repositories under the `Nebari-dev` organization.
```
Comment (kcpevey) [inline README.md]:
```suggestion
This is a meta repository that defines some shared files for the repositories under the `Nebari-dev` organization.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+always have the best reputation for being friendly and welcoming, but the Nebari team truly believes that everyone
+belongs in open source, and we are dedicated to making you feel welcome.
+
+All contributions are welcome, including issues, contributing code, new docs as well as updates and tweaks, blog posts,
+helping out people, organizing community events, working on accessibility and design items, and more.
+Continue reading to learn what the community can do for you and what you can do for the community.
+By contributing to open source projects you can connect with people, learn new skills, become a subject-matter expert,
+and apply all learnings to your own projects.
+
+Our detailed contribution guidelines can be found [on the Nebari's main documentation site][nebari-community].
```
Comment (kcpevey) [inline CONTRIBUTING.md]:
```suggestion
Our detailed contribution guidelines can be found [on Nebari's main documentation site][nebari-community].
```

File: .github/workflows/sync-issue-templates.yaml [hunk 3]
```diff
@@ -10,12 +10,14 @@ jobs:
...
-      - name: Checkout Repository
+      - name: Checkout Repository 🛎
         uses: actions/checkout@v3
 
-      - name: Run GitHub File Sync
+      - name: Run GitHub File Sync ♻️
         uses: BetaHuhn/repo-file-sync-action@v1
         with:
-          GH_PAT: ${{ secrets.ACCESS_TOKEN }}
+          GH_PAT: ${{ secrets.GH_PAT }}
```
Comment (kcpevey) [inline .github/workflows/sync-issue-templates.yaml:19]: can we add a comment here saying who this PAT belongs to? When something breaks its always frustrating to figure out who owns the PAT. And just my two cents, I think it should be `quansight-bot`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+### Any other comments?
+
+<!--
+Please be aware that we are a loose team of volunteers so patience is
+necessary; assistance handling other issues is very welcome. We value
+all user contributions. If we are slow to review, either the pull request needs some benchmarking, tinkering,
+convincing, etc. or more likely the reviewers are busy. In either case, we ask for your understanding during the
+review process.
+Thanks for contributing  to Nebari🙏🏼!
```
Comment (kcpevey) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Thanks for contributing to Nebari 🙏🏼!
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -98,6 +110,7 @@ body:
         - "Dask"
         - "dashboards"
         - "Grafana"
+        - "Argo"
```
Comment (kcpevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:113]: There is a typo on line 104 (just above here): `Is this issue related to onay of the Nebari integrations?`

**Review by kcpevey — APPROVED** [review]
Requested a few spelling/wording fixes. Once those are taken care of, feel free to merge!

File: .github/workflows/sync-issue-templates.yaml (hunk 3 above)
Comment (trallard) [reply .github/workflows/sync-issue-templates.yaml:19]: I am not sure who owns the `quansight-bot` or how to add a PAT from it so if you have a pointer do let me know
Right now I added a PAT I generated specifically for this
